### ISSUE TITLE
[✨]  Make qwik / eslint-plugin-qwik compatible with ESLint version 9

### ISSUE THREAD

**Issue by orbanszlrd** [QwikDev/qwik#6774]
### Is your feature request related to a problem?

qwik / eslint-plugin-qwik depends on an older version of ESLint and can't be used with the latest versions (9.0.0 and above). 
Please update the dependencies and migrate your config file. 
Thank you in advance!

### Describe the solution you'd like

1. Update the dependencies in the package.json:

-        "@types/eslint": "^8.56.10",
-        "@typescript-eslint/eslint-plugin": "^7.16.1",
-        "@typescript-eslint/parser": "^7.16.1",
-        "eslint": "^8.57.0",
+        "@types/eslint": "^9.6.0",
+        "@typescript-eslint/eslint-plugin": "^8.0.1",
+        "@typescript-eslint/parser": "^8.0.1",
+        "eslint": "^9.8.0",

2. Migrate the config file as described here: https://eslint.org/docs/latest/use/configure/migration-guide
4. Delete the .eslintignore as it's no longer needed

### Describe alternatives you've considered

no alternatives

### Additional context

![image](https://github.com/user-attachments/assets/f14a5378-57f9-4f2f-9590-16a7fb85c52f)

**Comment by gioboa** [comment]
Hi @orbanszlrd thanks for the report. As you can see [here](https://github.com/QwikDev/qwik/issues/6648) the Qwik team is looking to switching to [Biome](https://biomejs.dev/).

**Comment by orbanszlrd** [comment]
Yes, I saw that, but I suspect it's more work. When is the transition expected? Does this also mean that the ESLint update is not on the agenda?

**Comment by gioboa** [comment]
As a team we decided to switch to Biome and move the eslint as a community integration.
We are working on Biome integration, you can help us if you want btw

**Comment by orbanszlrd** [comment]
Thanks for the information, @gioboa !

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
